draft-documents/a-roadmap-for-ssi.md

```diff
@@ -1,346 +1,166 @@
+
+Specifically, this paper is targeted at the early adopters within a minimal viable community of stakeholders that share a common interest in the maturation of decentralized self-sovereign identity:
+
+* Incubation thought leaders at ground zero that are passionate about bringing Self-Sovereign Identity (SSI) to maturity.
+* Participants in open standards organizations 
+* Participants working on open source code based on those open standards
+* Early adopters within the developer community 
+* Change agents within the media, enterprise and public sectors
+* University and research institutes 
+* Forward thinking venture capital investors and incubators
 
--   Claire Rumore, FutureLab
+## Goals
+This paper is intended as a discussion starter. The community is mainly comprised of volunteers working remotely in their spare time. The objective is to help frame an approach that will place concentrated focus on key success factors and deliverables. Our discussion not only suggests best practices for businesses to create a SSI strategy but also helps to debunk some of the myths that have arisen due to a lack of understanding which of course is a direct result of our failure to simplify and articulate the concepts and benefits of SSI for the masses. We will touch on the necessity for toolkits, minimum viable environments (MVE), and interoperable end-to-end exemplars. Specifically, we suggest that attention must be placed on the following goals:
+
+* Outline a better approach for communicating a crisp, cohesive and comprehensive vision of the core benefits of SSI.
+* Justify the importance of streamlining the developer experience through tools that improve time to value and assist in the development of compelling applications.
+* Shift the focus from longer term ad-hoc brainstorming towards a formal initiative to develop a set of minimum viable reference applications.
+* Help find the WOW (the balance between low hanging fruit scenarios and instantly graspable killer app features) that inspires hope and stimulates media interest and viral propagation so that we can win the hearts and minds of a critical mass of stakeholders.
+
+## Success Metrics
+To address these goals, the community needs to crystalize a cohesive maturation roadmap (plan) that outlines initiatives which are measurable via key performance indicators. For example:
+
+* The community goal is to see one million DIDs published by March 2019
+* The release of a handful of reference applications that provide a down payment on the promise of decentralized identity
+* Formalized plans by year end 2018 for the foundational SSI specifications
+	* W3C DID
+	* W3C VC
+	* Oasis DKMS
+* Go to Market Resources to support developers by year end 2018
+	* Glossary of Terms
+	* Convergence of technical primers (tutorials) into a single  getting started kit
```

Comment:
Should add "Prepare a Media Kit that has a cohesive narrative about the initiative and its goals"

Reply:
done in new PR